scripts/workers/code_upload_submission_worker.py

```diff
@@ -95,8 +94,11 @@ def create_job_object(message, environment_image):
     init_container = client.V1Container(
         name="init-container",
         image="busybox",
```

Comment:
Should `ubuntu:latest` come here instead of `busybox`?

Reply:
Fixed it. Thanks for pointing it out.